src/server/announcements.ts

```diff
@@ -58,9 +58,17 @@ export class Announcements {
   }
 
   async announce (actor: string, info: ActorInfo): Promise<void> {
-    const existedAlready = await this.apsystem.store.actorsDb.get(actor)
+    let existedAlready = false
+    try {
+      const existingActor = await this.apsystem.store.actorsDb.get(actor)
+      if (existingActor === undefined) existedAlready = true
```

Comment:
Why would undefined mean that it exists already? Wouldn't it mean that it doesn't currently exist? Maybe we should catch both that and the level error and throw an error for both.